public/index.html

```diff
@@ -55,7 +55,7 @@
         </p>
         <button type="button" onclick="location.href='https://github.com/CogComp/cogcomp-nlp/tree/master/pipeline';" class="btn btn-danger btn-xs">Read More</button>
     </div>
-
+    <div class="loader" id="loader"></div>
```

Comment:
This is how it looks: 
<img width="930" alt="screen shot 2017-04-03 at 6 21 02 pm" src="https://cloud.githubusercontent.com/assets/2441454/24679623/62827a32-1953-11e7-986b-8f85328efa97.png">